backend/packages/worker/src/worker/parsing/storage.py

```
@@ -0,0 +1,35 @@
+import json
+import boto3
+from typing import Optional
+from botocore.exceptions import ClientError
+from .models import ParsedArtifact
+
+class ArtifactStorage:
+    def __init__(self, bucket_name: str, endpoint_url: Optional[str] = None):
```

Ruff (0.15.11)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)